### PR TITLE
Add Anthony Dahanne to CF contributors

### PR DESCRIPTION
**Pull request by anthonydahanne**
I am a new software engineer from VMware and am requesting to become a CF contributor.

### CODE REVIEW

**Review by Gerg — APPROVED** [review]
@anthonydahanne has made numerous contributions to Cloud Foundry projects, especially in to Paketo projects. They meet the [criteria for the Contributor rile](https://github.com/cloudfoundry/community/blob/e494db24e625f0ee0183009167a1783f24824864/toc/ROLES.md#contributor).

**Comment by Gerg** [comment]
Per the [Contributor promotion process](https://github.com/cloudfoundry/community/blob/e494db24e625f0ee0183009167a1783f24824864/toc/ROLES.md#promotion-to-contributor), you require a second review for this PR. Perhaps @pivotal-david-osullivan can provide?

**Review by pivotal-david-osullivan — APPROVED** [review]

**Pull request merged**
